### DISCUSSION TITLE
Exporting Training Metrics Data

### DISCUSSION THREAD

**Discussion by clee758** [talmolab/sleap#1353] [Help!]
Hi, I performed multiple trainings and predictions on one video file, and I was wondering if there was a way to export the training metric data for each centroid and centered instance model in one compiled data set, to compare the accuracy between predictions. Thank you!

**Comment by roomrys** [comment]
Hi @clee758,

SLEAP saves the metrics for each model in a metrics.train.npz file. This file is located in the models directory and then under the folder for the specific model you want the metrics for, i.e. `models\230614_221003.centered_instance.n=101`. 

The metrics calculated will be for all the annotations in your project at the time of training. These will not be compiled in one dataset by default, but all the information should be there for you to create that unified dataset. Here is a [notebook on model evaluation](https://sleap.ai/notebooks/Model_evaluation.html) that should be helpful!

Thanks,
Liezl

**Comment by clee758** [comment]
Hi, do I have to move the data to Google Colab to create the unified dataset? I believe the notebook was giving Colab specific instructions. Thanks!

**Reply by roomrys** [reply]
Hi @clee758,

If you would prefer, you can use the code provided in the notebook, copy (and modify) it to a local script and run that script locally to access your models folder. We like to use notebooks for quick iteration and sharing, but it is up to your preference!

Thanks,
Liezl
